components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -171,11 +133,14 @@ function ConversationList(props: ConversationListProps) {
                 <div
                   className={styles.message}
                 >
-                  {!turn.selectedText &&
-                      <FormattedTextRenderer text={turn.text} />}
+                  {!turn.selectedText ? (
+                    isAIAssistant
+                      ? <MarkdownRenderer text={turn.text} shouldShowTextCursor={shouldShowTextCursor} />
+                      : turn.text
+                  ) : null // If there is selectedText, don't render turn.text at all
+                  }
                   {turn.selectedText &&
                       <ActionTypeLabel actionType={turn.actionType} />}
-                  {isLoading && <span className={styles.caret} />}
```

Comment:
Should we remove `caret` from the style too:?

Reply:
removed